lib/dialects/abstract/query-generator.js

```diff
@@ -1519,8 +1535,26 @@ const QueryGenerator = {
 
         mainQueryOrder.push(this.quote(t, model, false, '->'));
       }
+    } else if (typeof options.order === 'string') {
+      // split
+      for (let order of options.order.split(',')) {
+        // get options
+        const orderOption = getOrderOption(order);
+        if(orderOption.index > -1){
+          order = order.substring(0, orderOption.index);
+        }
+        order = _.trim(order);
+        let sql = this.quote(order, model, false, '->');
+        if(orderOption.option){
+          sql += ' ' + orderOption.option;
+        }
+        if (subQuery) {
+          subQueryOrder.push(sql);
+        }
+        mainQueryOrder.push(sql);
+      }
```

Comment:
@felixfbecker Understood.  Will modify the tests/logic/documentation to only accept array.  Then I will do `group` option then.